lib/ext/heroku/command/addons.rb

```diff
@@ -174,7 +174,14 @@ def create
 
       action("Creating #{addon['name'].downcase}") {}
       action("Adding #{addon['name'].downcase} to #{app}") {}
-      action("Setting #{addon['config_vars'].join(', ')} and restarting #{app}") do
+
+      if addon.has_key? 'config_vars' && addon['config_vars']
```

Comment:
This seems to have trickled out. @mathias do you want somebody to take this off your hands?